### PR TITLE
[Bug] Unique error on schedule call, [Bug] Undefined error on register

### LINKED ISSUE #63: [Register] Internal server error on register
**Describe the bug**
Internal server error on register, `TypeError: Cannot read property 'profileImage' of undefined
    at ResponseDtoService.buildUserRO (C:\Users\mpek6\Desktop\Ameelio\connect-api-nest\dist\common\services\ResponseDtoService.js:30:38)`

**To Reproduce**
RegisterError2

**Expected behavior**
Registers the user, returning the successfully created user object with a valid cookie

### PR DESCRIPTION
**Pull request by mpekala23**
## Proposed changes
Fixed two quick errors. The unique check in the schedule call function was failing (returning false even when items were indeed unique) and the `person` object in the user response from register was being read when it was undefined.

## Related Issues
Closes #63 

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: src/calls/services/call.service.ts [hunk 1]
```diff
@@ -74,8 +74,8 @@ export default class CallsService extends CrudService<Call> {
 
   private areAllParticipantsUnique(inmateIds: number[], userIds: number[]) {
     if (
-      inmateIds.some((inmateId, i) => inmateIds.indexOf(inmateId) !== i) ||
-      userIds.some((userId, i) => inmateIds.indexOf(userId) !== i)
+      inmateIds.some((inmateId, i, arr) => arr.indexOf(inmateId) !== i) ||
```
Comment (gabrielsaruhashi) [inline src/calls/services/call.service.ts:77]: wait why is this set up like this? ccan u send me a sample 

what is this arr?

**Review by gabrielsaruhashi — CHANGES_REQUESTED** [review]
some nits!

File: src/calls/services/call.service.ts [hunk 2]
```diff
@@ -75,7 +75,7 @@ export default class CallsService extends CrudService<Call> {
   private areAllParticipantsUnique(inmateIds: number[], userIds: number[]) {
     if (
       inmateIds.some((inmateId, i) => inmateIds.indexOf(inmateId) !== i) ||
-      userIds.some((userId, i) => inmateIds.indexOf(userId) !== i)
+      userIds.some((userId, i) => userIds.indexOf(userId) !== i)
```
Comment (mpekala23) [inline src/calls/services/call.service.ts:78]: @gabrielsaruhashi this is the change that should've been made, the `arr` solution worked as well (arr was just a copy of whatever array you were iteratng on)

**Review by gabrielsaruhashi — APPROVED** [review]
lgtm! congrats on ur PR #2!

**Pull request merged**